memflow/src/types/address.rs

```diff
@@ -82,6 +82,22 @@ impl Address {
         Address::NULL
     }
 
+    /// Returns an address with input value.
+    ///
+    /// Useful for defining constants
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use memflow::types::Address;
+    ///
+    /// const TEN: Address = Address::from_u64(10)
+    /// ```
+    #[inline]
+    pub const fn from_u64(item: u64) -> Self {
```

Comment:
This can also be written via 10.into() but I guess a separate function for this does not hurt.